src/components/ui/PaymentRequestActionMenu/PaymentRequestActionMenu.tsx

```diff
@@ -55,6 +57,7 @@ const PaymentRequestActionMenu = ({ onDuplicate, onCopyLink, onDelete, onBlur }:
   const onDuplicateClick = (e: React.MouseEvent<HTMLDivElement>) => handleClick(e, onDuplicate);
   const onCopyLinkClick = (e: React.MouseEvent<HTMLDivElement>) => handleClick(e, onCopyLink);
   const onDeleteClick = (e: React.MouseEvent<HTMLDivElement>) => handleClick(e, onDelete);
+  const emptyClick = (e: React.MouseEvent<HTMLDivElement>) => e.stopPropagation();
```

Comment:
Not sure but the tailwind class 'pointer-events-none' might work instead?

Reply:
Tried it, but the click event still propagates.